decorator.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'nameable'
+
+class Decorator < Nameable
+  def initialize(nameable)
+    super
+    @namable = nameable
+  end
+
+  def correct_name
+    @nameable
```

Comment:
- [ ] For this, you are supposed to return the result of the correct_name method of the @nameable, which means you have to also call the correct_name on it. this should be like this: ```@nameable.correct_name```

![image](https://user-images.githubusercontent.com/86318284/199222335-8d41cca8-6630-477a-901a-ebe15963a564.png)